docs/quick-start.md

```diff
@@ -36,6 +36,18 @@ The second command will download and run a prebuilt AsterixDB docker container f
 After it finishes, you should see the messages as shown in the following screenshot:
 ![docker][docker]
 
+After you have installed the docker container for the first time, you may restart it later using the following command:
```

Comment:
For an end-user of Cloudberry, it is still common that he/she installed the AsterixDB/TwitterMap for the first time. After the machine is shutdown, what should be done then? Definitely he shouldn't re-install docker/AsterixDB again. I think such information is needed, but maybe put it at other places should also be fine.